app/views/components/_related-navigation.html.erb

```diff
@@ -1,4 +1,5 @@
 <%
+  max_section_length = 5
```

Comment:
Could we show more than 5? Is this based on the example posted, could we show more by default?

Reply:
It's based on some data analysis @steventux did on how many countries tend to be displayed (as that was the biggest issue we found) - I don't have the figures to hand as they've been lost on Slack. Also, the metadata component in static limits at 5, so it seemed sensible to follow suit.